doc/ROADMAP.rst.txt

```diff
@@ -0,0 +1,377 @@
+Scipy Roadmap
+=============
+
+Most of this roadmap is intended to provide a high-level view on what is
+needed per scipy submodule in terms of new functionality, bug fixes, etc.
+before we can release a ``1.0`` version of Scipy.  Furthermore it contains
+ideas for major new features - those are marked as such, and are not needed
+for Scipy to become 1.0.  Things not mentioned in this roadmap are
+not necessarily unimportant or out of scope, however we (the Scipy developers)
+want to provide to our users and contributors a clear picture of where Scipy is
+going and where help is needed most urgently.
+
+When a module is in a 1.0-ready state, it means that it has the functionality
+we consider essential and has an API and code quality (including documentation
+and tests) that's of high enough quality.  
+
+
+General
+-------
+This roadmap will be evolving together with Scipy.  Updates can be submitted as
+pull requests and, unless they're very minor, have to be discussed on the
+scipy-dev mailing list.
```

Comment:
> I'd say open a PR, then if it's deemed to be too disruptive someone will suggest to discuss on the mailing list before merging

Sounds good to me. But we might want to say something along the lines that for larger PRs people might want to start a discussion first in case the proposal is substantially altered (or rejected) so that they don't waste coding time.